subsurface-mobile-main.cpp

```diff
@@ -88,10 +88,9 @@ int main(int argc, char **argv)
 
 void set_non_bt_addresses()
 {
-#if SERIAL_FTDI
-	connectionListModel.addAddress("FTDI");
-#endif
 #if defined(Q_OS_ANDROID)
+    connectionListModel.addAddress("usb-serial-for-android");
+	connectionListModel.addAddress("FTDI");
```

Comment:
see my comment in email - is that still useful? This fails for almost all Android devices and should be fully replaced by your code, right?

Reply:
fixed

Reply:
See mail: two dive computers need set_break functionality not present in usb-serial-for-android